tests/fixtures/tests.py

```diff
@@ -532,6 +532,16 @@ def test_compressed_specified_loading(self):
         self.assertQuerysetEqual(Article.objects.all(), [
             '<Article: WoW subscribers now outnumber readers>',
         ])
+        Article.objects.all().delete()
+        management.call_command('loaddata', 'fixture5.json.gz', verbosity=0)
+        self.assertQuerysetEqual(Article.objects.all(), [
+            '<Article: WoW subscribers now outnumber readers>',
+        ])
+        Article.objects.all().delete()
+        management.call_command('loaddata', 'fixture5.json.bz2', verbosity=0)
+        self.assertQuerysetEqual(Article.objects.all(), [
+            '<Article: WoW subscribers now outnumber readers>',
+        ])
```

Comment:
Split into three tests rather than deleting the data:

```suggestion
    def test_compressed_loading_gzip(self):
        # Load fixture 5 (compressed, gzip), using format *and* compression specification
        management.call_command('loaddata', 'fixture5.json.gz', verbosity=0)
        self.assertQuerysetEqual(Article.objects.all(), [
            '<Article: WoW subscribers now outnumber readers>',
        ])
    def test_compressed_loading_bzip(self):    
        # Load fixture 5 (compressed, bzip), using format *and* compression specification
        management.call_command('loaddata', 'fixture5.json.bz2', verbosity=0)
        self.assertQuerysetEqual(Article.objects.all(), [
            '<Article: WoW subscribers now outnumber readers>',
        ])
```

Then they can fail indepenently.

Reply:
Ok, thanks for your feedback